src/grpc/infer_handler.h

```diff
@@ -1529,7 +1568,7 @@ class ModelInferHandler
 
  protected:
   void StartNewRequest() override;
-  bool Process(State* state, bool rpc_ok) override;
+  bool Process(State* state, bool rpc_ok, bool is_notification) override;
```

Comment:
for parity with stream_infer_handler, let's set `is_notification` by default to false

Reply:
`Process` is called at one place in [infer_handler.h](https://github.com/triton-inference-server/server/blob/cd92b058ec99522cd61bdd4368eade0d8b556891/src/grpc/infer_handler.h#L1448) where `is_notification` is always passed. The default parameter is never used in `Process`. Do you still think we should make this change?